client/src/components/Post.tsx

```diff
@@ -1,26 +1,51 @@
-import React, { ReactElement, useState } from 'react';
-import { PostWithRelations } from '../../../types';
-import { Link } from 'react-router-dom';
-import MarkDown from './MarkDown';
+import React, {ReactElement, useState} from "react";
+import { PostWithRelations } from "../../../types";
+import { Link } from "react-router-dom";
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+dayjs.extend(relativeTime);
+
+import MarkDown from "./MarkDown";
 
 import Card from '@mui/material/Card';
 import Avatar from '@mui/material/Avatar';
-import Box from '@mui/material/Box';
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography';
+import IconButton from '@mui/material/IconButton';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import CommentIcon from '@mui/icons-material/Comment';
+
+const Post = ({content} : {content: PostWithRelations}): ReactElement => {
+
+  const [like, setLike] = useState(false);
+
+  const handleLike = () => {
+    setLike(!like);
+  }
 
-const Post = ({ content }: { content: PostWithRelations }): ReactElement => {
   return (
-    <Card>
-      <Box sx={{ display: 'flex', flexDirection: 'row' }}>
+    <Card sx={{marginBottom: 5}}>
+      <Box sx={{display:"flex", flexDirection:'row', marginLeft: 1, marginTop: 1, alignItems: 'center'}}>
         <Link to={`/user/${content.author.id}/profile`}>
           <Avatar alt={content.author.username!} src={content.author.picture!}>
-            {content.author.username!}
-          </Avatar>
+            {'?'}

```

Comment:
Kinda curious about this, is this a placeholder for if there's no picture?

Reply:
![image](https://github.com/undefined-ims-india/mkdev/assets/119898626/66c82b0d-f556-4179-b706-62cb786e86ce)
It looks like this, I originally had it as the first letter of the username but that was causing problems with the username being null